### PR TITLE
Add table of contents + useful command section in README

### PR DESCRIPTION
**Pull request by waliens**

### CODE REVIEW

**Comment by waliens** [comment]
If it can be of any help, I have also developed a python script to display a summary of resources currently used (and/or awaited for). It uses ´sinfo´ and ´squeue´ as source of information. It produces tables such as this one: 

```log
Resources per node
         | jobs |       mem       |     cpus    |    gpus
-------- | ---- | --------------- | ----------- | ----------
 node-01 |    3 |  12G  257G   5% |  3  32   9% |  3  4  75%
 node-02 |    4 |  32G  257G  12% |  4  32  12% |  4  4 100%
 node-03 |    4 |  30G  257G  12% |  4  32  12% |  4  4 100%
 node-04 |    4 |  20G  257G   8% |  4  32  12% |  4  4 100%
 node-05 |    4 |  62G   64G  97% |  4  32  12% |  4  4 100%
-------- | ---- | --------------- | ----------- | ----------
  total: |   19 | 156G 1094G  14% | 19 160  12% | 19 20  95%
```

Here it is:
[queue_stats.zip](https://github.com/montefiore-ai/alan-cluster/files/3161402/queue_stats.zip)

**Comment by glouppe** [comment]
Nice script! Would be handy to have this by default indeed.

**Comment by waliens** [comment]
Thanks ! I have cleaned/documented the script a little bit and added command-line parameters.

**Comment by JoeriHermans** [comment]
This is quite nice, maybe we could integrate it as a default binary or something (per Gilles' suggestion). For instance, `atop` or `alantop`.

**Comment by JoeriHermans** [comment]
I would create a different repository for this particular tool in montefiore-ai though. Merging it with this particular repository would not really be ideal, as it is mainly docs etc. That way we can propose fixes or features for that particular shell-tool. Let me know what you think @waliens.

**Comment by waliens** [comment]
Good idea, didn't really where to put the script actually !

**Comment by JoeriHermans** [comment]
Ok then, can you create the repo @waliens ? Then we'll modify the README accordingly.

**Comment by waliens** [comment]
Done 👍

File: README.md [hunk 1]
```diff
@@ -119,6 +125,15 @@ TODO
 
 TODO
 
+## Cluster usage
+
+### Useful `slurm` commands
```
Comment (glouppe) [inline README.md]: Since you started, maybe we should properly document the main commands? sbatch, squeue, sinfo, scontrol?

Comment (JoeriHermans) [reply README.md]: Maybe we should point to the CECI documentation, this is quite well done imho https://support.ceci-hpc.be/doc/_contents/QuickStart/SubmittingJobs/SlurmTutorial.html

Comment (glouppe) [reply README.md]: Yes, we could add a pointer to that.

Then have some short explanations regarding how to request GPUs?

Comment (waliens) [reply README.md]: As the script I have made will eventually become a command, I have removed the squeue example and added links to CECI guide and slurm documentation for a handful of commands (+ a note about gpu request)

**Comment by glouppe** [comment]
Thanks! merged :)

**Pull request merged**